apps/api/SurveyResponses/survey_responses.py

```diff
@@ -0,0 +1,96 @@
+from django.http import HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+from jurisdiction.models import State, Jurisdiction
+from .check_authorization import checkAuth
+
+import json
+
+@csrf_exempt
+def GetSurveyResponse(request):
+    # check authorization
+    try:
+        assert checkAuth(request)
+    except:
+        response = HttpResponse()
+        response.status_code = 401
+        return response
+
+    decoded = request.body.decode('utf-8')
+    try:
+        json_dict = json.loads(decoded)
+    except json.decoder.JSONDecodeError:
+        return HttpResponse(status=400)
+    
+    print(json_dict)
+
+    # Find jurisdiction to update
+    jurisdiction_id = json_dict["Custom Variable__JurisdictionNo"]
+    j = Jurisdiction.objects.get(pk=jurisdiction_id)
+
+    for q, a in json_dict.items():
+        try:
+            q_list = q.split(')')
+            q_no = int(q_list[0])
+            # Sanitize input
+            a = str(a)
+        except ValueError: # not a question
+            if q_list[0][13:20] == "further":
+                # manually set "further notes"
+                print("Manually set q_no")
+                q_no = 13
+            else:
+                print("ValueError on '{}'".format(q_list[0]))
+                continue
```

Comment:
I think we should have a way to track responses that are failed to stored because there is an error